### PR TITLE
Option to show resolution only

### PR DESCRIPTION
**Pull request by xhoxye**
I don't know if this modification is in line with your original project intention, it may just be suitable for my use
If you merge, you can delete extraneous labels after merging
Stable Diffusion WebUI Resolutions selecto UI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,50 @@
-# Stable Diffusion WebUI Aspect Ratio selector
+# Stable Diffusion WebUI 分辨率选择器-xhox
+# Stable Diffusion WebUI Aspect Ratio Resolutions selector
 
-Extension for [AUTOMATIC1111/stable-diffusion-webui](https://github.com/AUTOMATIC1111/stable-diffusion-webui.git) adding image aspect ratio selector buttons.
+为 [lllyasviel/stable-diffusion-webui-forge](https://github.com/lllyasviel/stable-diffusion-webui-forge.git)、[AUTOMATIC1111/stable-diffusion-webui](https://github.com/AUTOMATIC1111/stable-diffusion-webui.git) 添加图片分辨率选择按钮。
```
Comment (alemelis) [inline README.md:4]: I'd rather keep the original links throughout the README

**Review by alemelis — COMMENTED** [review]
like the new feature but please revert the urls to the original ones

**Comment by xhoxye** [comment]
I don't know how to change it, you can merge it and change it yourself, I have no problem.

**Comment by xhoxye** [comment]
Or I'll push another branch.

**Pull request closed without merging**